app/src/main/res/layout/fragment_w_o_nav_comp_sec.xml

```diff
@@ -0,0 +1,251 @@
+        android:text="@string/today" />
+
+    <ImageView
+        android:id="@+id/weather_condition_img"
+        android:layout_width="@dimen/dp_64"
+        android:layout_height="@dimen/dp_64"
+        android:layout_marginBottom="@dimen/dp_32"
+        app:layout_constraintBottom_toTopOf="@+id/text_view_weather"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        android:src="@drawable/ic_sunny" />
+
+    <TextView
+        android:id="@+id/text_view_weather"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textColor="@color/black"
+        android:textSize="@dimen/sp_64"
+        android:textStyle="bold"
+        app:layout_constraintBottom_toTopOf="@+id/image_view410"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/text_view3"
+        tools:text="@string/temperature" />
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textColor="@color/black"
+        android:textSize="@dimen/sp_64"
+        android:textStyle="bold"
+        app:layout_constraintBottom_toBottomOf="@+id/text_view_weather"
+        app:layout_constraintStart_toEndOf="@+id/text_view_weather"
+        app:layout_constraintTop_toTopOf="@+id/text_view_weather"
+        tools:text="@string/cantigrate" />
+
+    <TextView
+        android:id="@+id/weather_condition"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/dp_16"
+        android:textColor="@color/black"
+        android:textSize="@dimen/sp_16"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/text_view_weather"
+        tools:text="@string/weather_condition" />
+
+    <TextView
+        android:id="@+id/wind"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/dp_32"
+        android:textColor="@color/black"
+        android:textSize="@dimen/sp_14"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/weather_condition"
+        android:text="@string/wind" />
+
+    <LinearLayout
+        android:id="@+id/l_layout"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/dp_8"
+        android:gravity="center"
+        android:orientation="horizontal"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/wind">
+
+
+        <ImageView
```

Comment:
Bu ImageView, TextView'in soluna gorseli yerlestirmek icin kullanildi sanirim. Bu sekilde yeni bir view kullanmaktansa TextView'e `app:drawableStartCompat="@drawable/ic_wind"` attribute gecilerek fazladan ekledigin viewlerden kurtulmus olursun

Reply:
Abi baya iyimiş bu özellik teşekkür ederim :) aralarına margin atma durumunu çözemedim ama yapışık oldular ne yapmam gerekiyor bunun için